test/storage/file.js

```diff
@@ -1025,6 +1025,22 @@ describe('File', function() {
     });
   });
 
+  describe('makePublic', function() {
+    it('should make the file public', function(done) {
+
+      // If this mocked method (THAT DOES NOTHING!) is removed, nothing works.
+      file.acl.makeAclObject_ = function() {};
```

Comment:
Is this a side-effect of using mockery? I had a feeling I should have been doing that instead. I'll update it to do that.